files/plugins/check_cis_audit.py

```diff
@@ -34,8 +34,10 @@ def _get_major_version():
     raise OSError("No VERSION_ID in /etc/os-release")
 
 
-# cis-audit changed between bionic and focal
-if _get_major_version() < 20:
+# cis audit changed from bionic to focal.
+DISTRO_VERSION = _get_major_version()
+
+if DISTRO_VERSION < 20:
```

Comment:
nit: I think this can be left unchanged to reduce noise :)  We can refactor it later.